src/adapters/v00/marketplace_adapter.js

```diff
@@ -265,6 +267,8 @@ class MarkeplaceAdapter {
       if (e.event === 'OfferData') {
         rawOffer.status = '4'
       }
+      // Translate status number to string
+      rawOffer.status = OFFER_STATUS[rawOffer.status]
```

Comment:
Will this require updates to the DApp ?
If yes, let's make sure to land both this PR and the companion DApp PR at about the same time so that we don't break the DApp for developers for too long...